.gitignore

```diff
@@ -41,6 +41,9 @@ build/Release
 node_modules/
 jspm_packages/
 
+# Uploads directory for users
+uploads/
```

Comment:
If the folder is added to gitignore, then why is the uploaded image still committed? Look into this.